### PR TITLE
Player names scores

### PR DESCRIPTION
**Pull request by MHOAKEY**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,6 +27,39 @@
     <div id="text"></div>
     <div><br /><button id="restart">RESTART</button></div>
     <footer>
+      <br />
```
Comment (oakeef) [inline index.html:30]: usually the footer would be for things that belong at the very bottom of a page
I would probably move all this form stuff/score stuff above the footer in it's own `<div>`

File: index.html [hunk 2]
```diff
@@ -27,6 +27,39 @@
...
+      <form>
+        <label id="P1Name">Player X:</label>
```
Comment (oakeef) [inline index.html]: I don't think you need to give the label an `id`
I'm not sure you need to target it for anything
but there is a `for` property that should match the `id` value of the form element it is a label for

example: https://www.w3schools.com/tags/tag_label.asp

File: script.js [hunk 3]
```diff
@@ -16,12 +20,21 @@ const restart = document.getElementById("restart");
 restart.addEventListener("click", clearGameBoard);
 gameSpaces.forEach(addEventListenerToGameSpaces);
 
+document.addEventListener("keydown", enterKeyDisable);
+
+text.innerHTML = "Player X turn";
+
+function enterKeyDisable(event) {
+  if (event.keyCode == "13") {
+    console.log("disableEnter");
```
Comment (oakeef) [inline script.js]: you can remove this `console.log()` now

File: script.js [hunk 4]
```diff
@@ -119,13 +132,36 @@ function clearGameBoard(event) {
...
+}
+
+function insertPlayerOName(userInput) {
+  let p2Name = document.getElementById("P2Name");
+  let input2Name = document.getElementById("P2NameInput").value;
+
+  p2Name.innerHTML = input2Name;
+}
+
+function keepScore(gamePiece) {
```
Comment (oakeef) [inline script.js]: nit pick but maybe `updateScore` is a better name for this function?

File: script.js [hunk 5]
```diff
@@ -5,9 +5,13 @@
 // - A User should not be able to select a box that is already selected
 // - When game is over (whether a win or tie) the users should be prompted to play again and the game should be reset
 
+//event.preventdefault
```
Comment (oakeef) [inline script.js]: you can remove this

**Review by kfoley123 — APPROVED** [review]

File: script.js [hunk 6]
```diff
@@ -119,13 +131,40 @@ function clearGameBoard(event) {
   disableGameSpaces(false);
 }
 
+function insertPlayerXName(userInput) {
+  let p1Name = document.getElementById("P1Name");
+  let inputName = document.getElementById("P1NameInput").value;
+
+  p1Name.innerHTML = inputName;
+  document.getElementById("P1NameInput").style.visibility = "hidden";
```
Comment (oakeef) [inline script.js]: you already have this element saved in a variable called `inputName` from above so you can replace all the
`document.getElementByID("P1NameInput)` with `inputName`

File: script.js [hunk 7]
```diff
@@ -119,13 +131,40 @@ function clearGameBoard(event) {
...
+  document.getElementById("bt").style.visibility = "hidden";
+}
+
+function insertPlayerOName(userInput) {
+  let p2Name = document.getElementById("P2Name");
+  let input2Name = document.getElementById("P2NameInput").value;
+
+  p2Name.innerHTML = input2Name;
+  document.getElementById("P2NameInput").style.visibility = "hidden";
```
Comment (oakeef) [inline script.js]: same as above, but `input2Name`

**Review by oakeef — APPROVED** [review]

**Pull request merged**